src/TargetingData.ts

```diff
@@ -118,7 +123,7 @@ export const TargetingData = {
     },
 
     /**
-     * Sets the phone number of the user.  Sets null to clear this value.
+     * Sets the phone number of the user.  Set this to null to clear this value.
```

Comment:
Can we conform to same styling as the API docs of all of our other plugins? That is, we use one space, and not two spaces, after each period.